src/GeneralStateTestsFiller/stEIP3670/CreateTransactionInvalidEOF1Filler.yml

```diff
@@ -0,0 +1,1462 @@
+      # mstore(0x00, 0xef000101001b0060016001550000000000000000000000000000000000000000) return(0x00, 0x0d) push29(
+      - ':label truncated_push29_a :raw 0xef000101002a007fef000101001b0060016001550000000000000000000000000000000000000000600052600d6000f37c'
+      # mstore(0x00, 0xef000101001b0060016001550000000000000000000000000000000000000000) return(0x00, 0x0d) push29(0x00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fd
+      - ':label truncated_push29_b :raw 0xef0001010046007fef000101001b0060016001550000000000000000000000000000000000000000600052600d6000f37c00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fd'
+      # mstore(0x00, 0xef000101001b0060016001550000000000000000000000000000000000000000) return(0x00, 0x0d) push30(
+      - ':label truncated_push30_a :raw 0xef000101002a007fef000101001b0060016001550000000000000000000000000000000000000000600052600d6000f37d'
+      # mstore(0x00, 0xef000101001b0060016001550000000000000000000000000000000000000000) return(0x00, 0x0d) push30(0x00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfe
+      - ':label truncated_push30_b :raw 0xef0001010047007fef000101001b0060016001550000000000000000000000000000000000000000600052600d6000f37d00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfe'
+      # mstore(0x00, 0xef000101001b0060016001550000000000000000000000000000000000000000) return(0x00, 0x0d) push31(
+      - ':label truncated_push31_a :raw 0xef000101002a007fef000101001b0060016001550000000000000000000000000000000000000000600052600d6000f37e'
+      # mstore(0x00, 0xef000101001b0060016001550000000000000000000000000000000000000000) return(0x00, 0x0d) push31(0x00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff
+      - ':label truncated_push31_b :raw 0xef0001010048007fef000101001b0060016001550000000000000000000000000000000000000000600052600d6000f37e00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff'
+      # mstore(0x00, 0xef000101001b0060016001550000000000000000000000000000000000000000) return(0x00, 0x0d) push32(
+      - ':label truncated_push32_a :raw 0xef000101002a007fef000101001b0060016001550000000000000000000000000000000000000000600052600d6000f37f'
+      # mstore(0x00, 0xef000101001b0060016001550000000000000000000000000000000000000000) return(0x00, 0x0d) push32(0x00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00
+      - ':label truncated_push32_b :raw 0xef0001010049007fef000101001b0060016001550000000000000000000000000000000000000000600052600d6000f37f00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00f3fdfeff00'
+
+      # Invalid EOF Init code: Containing undefined instruction after STOP
+      # mstore(0x00, 0xef000101001b0060016001550000000000000000000000000000000000000000) stop  0xf6'
+      - ':label undef_instr_after_stop :raw 0xef0001010026007fef000101001b006001600155000000000000000000000000000000000000000060005200f6'
+
+      # Invalid EOF Init code: Containing non terminating instruction at the end
```

Comment:
I think we can remove this section now.

These cases are invalid but for other reasons - not enough arguments on stack and unreachable code anyway - but these should be tested separately in 5450 tests.

Reply:
removed